gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/JavaServiceDistributionPlugin.java

```diff
@@ -260,12 +260,61 @@ public void execute(Task _task) {
             }
         }));
 
-        project.afterEvaluate(p -> DistTarTask.configure(
-                distTar.get(),
-                project,
-                distributionExtension.getDistributionServiceName().get(),
-                distributionExtension.getExcludeFromVar().get(),
-                distributionExtension.getEnableManifestClasspath().get()));
+        project.afterEvaluate(proj -> distTar.configure(task -> {
```

Comment:
awesome to get rid of a bit more groovy, but could we keep this separate just to keep the plugin clean

Reply:
moved